Gemfile

```diff
@@ -17,7 +17,8 @@ gem "terser"
 gem "gds-api-adapters"
 gem "gds-sso"
 gem "govuk_app_config"
-gem "govuk_publishing_components"
+# gem "govuk_publishing_components"
+gem "govuk_publishing_components", "40.0.0"
```

Comment:
We can probably just revert this line back to `gem "govuk_publishing_components"` now, can't we?

Reply:
👍 Yes, I have changed that to match the equivalent in Publisher.